### ISSUE TITLE
Build fails on mac: YbridOgg.framework: bundle format is ambiguous (could be app or framework)

### ISSUE THREAD

**Issue by krzysiek84** [ybrid/ogg-swift#1]
Hi, First of all - thank you for sharing the swift wrapper for ogg!
I was trying to use it via pods in my project, but when trying to build it I'm getting the following error:

```

rsync --delete -av --filter P .*.?????? --links --filter "- CVS/" --filter "- .svn/" --filter "- .git/" --filter "- .hg/" --filter "- Headers" --filter "- PrivateHeaders" --filter "- Modules" "/Path/TO/Products/Debug/XCFrameworkIntermediates/YbridOgg/YbridOgg.framework" "/Path/To/App/Contents/Frameworks"

building file list ... done

YbridOgg.framework/
YbridOgg.framework/YbridOgg

sent 188533 bytes  received 48 bytes  377162.00 bytes/sec
total size is 586044  speedup is 3.11
Stripped /Path/To/App/Contents/Frameworks/YbridOgg.framework/YbridOgg of architectures: arm64

/usr/bin/codesign --force --sign XXX  --preserve-metadata=identifier,entitlements '/path/to/App/Contents/Frameworks/YbridOgg.framework'
/path/to/App/Frameworks/YbridOgg.framework: replacing existing signature
/path/to/App/Contents/Frameworks/YbridOgg.framework: bundle format is ambiguous (could be app or framework)
````

Update:
When building the xcframework locally:
```
./prepare
./build
```

And adding the framework to project, it works fine.

**Comment by FlorianAtNacamar** [comment]
Hi. Thanks for the bug report. I can reproduce the error in a workspace using YbridOgg and I can see problem when I select ‘My Mac’ as the target device. I guess there is something wrong with the signature for mac devices.

Could you please tell me whether ogg-swift/test.sh passes the tests for your mac (device ‘My Mac’) and for target ‘Any Mac’?
And in addition for which use case do you consider to use our framework?

**Comment by krzysiek84** [comment]
Hi, I had to change the codesigning to my "team". And then the tests succeeded.

But not sure if it's related to code signing,  was thinking it's something with symbolic links: https://blog.inventic.eu/2015/03/os-x-codesign-failed-bundle-format-is-ambiguous-could-be-app-or-framework/

I've created a PR with some fixes, but canceled it as gave up with trying to include Ogg support :/
https://github.com/ybrid/ogg-swift/pull/2

**Comment by FlorianAtNacamar** [comment]
I did not recognize the aborted Pull-Request until now! I will care to recieve more notifications in future. Yes, sed is not the most compatible tool of the world. I should improve prepare.sh too.

If it's not the signing, maybe something happens to the symbolic links when zipping the xcframework in the end of build.sh, before uploading it. I will check this as well. Thanks in advance.

**Comment by FlorianAtNacamar** [comment]
Symbolic links have to be retained in the zipped framework, see [developer.apple.com/.../Concepts/FrameworkAnatomy.html](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/FrameworkAnatomy.html). If not, [others](https://github.com/nferruzzi/openssl) have also reported code signings problems. I fixed this in build.sh by zipping with -y and did a bux fix release 0.7.4. 
I'm closing the issue but please reopen if you run into this again.
